### ISSUE TITLE
[ENHANCEMENT] NISRA Web App Accessibility Audit Recommended Enhancements

### ISSUE THREAD

**Issue by CHanamy57** [CSOIreland/PxStat#606]
Enhancements based on Failed features or Recommendations from NISRA after Accessibility Testing

**Comment by CHanamy57** [comment]
Empty Link - Social Media Links
Issue: The social media links in the page footer have no text
meaning users with screen readers have no clue as to
where these links are going.

Fix: Aria Labels added to config data for each link - link's action is now announced by screen reader

**Comment by CHanamy57** [comment]
Broken ARIA Reference:
Issue: An aria labelledby or aria describedby reference exists, but the target for the reference does not exist.

Fix: Remove redundant "aria-describedby" or "aria-labelledby" references where "aria-label" is present for screen reader & no targets exist.

**Comment by CHanamy57** [comment]
Insufficient Labelling
Issue: Arrows announced by screen reader should have appropriate label

Fix: Arrows have aria-label of _eg_: **"TLIST(A1)Year: activate to sort column ascending"**/ **"TLIST(A1)Year: activate to sort column descending"**

**Comment by CHanamy57** [comment]
Status Messages
Issue: No status message after search input to indicate how many results are returned (in Data search entity).

Recommended fix: Include a way of informing the user on the amount of results.

**Comment by CHanamy57** [comment]
Contrast Issues
Issue: Interface elements contain illegible text on hover

Recommended fix: Change hover state of Navigation Buttons from grey text to lighter color/ Update hover state of elements

**Comment by CHanamy57** [comment]
Missing Form Labels
Issue: Insufficient labelling for check boxes - eg: Browse//Languages - Icon is used instead of Bootstrap Checkbox. 

Recommended fix: Replace checkbox icons with Bootstrap Checkbox to for allow proper labelling on check & uncheck.

**Comment by liamie** [comment]
Enhancements completed
